### ISSUE TITLE
Unexpected error in rest_quickstart.ipynb

### ISSUE THREAD

**Issue by Ghat0tkach** [google/generative-ai-docs#208]
### Description of the bug:

When running in the google collab notebook , every bash command is giving the following error
![image](https://github.com/google/generative-ai-docs/assets/59855919/a952b8ff-3a72-4e80-b7bd-813f4fec36bb)

### Actual vs expected behavior:

There should be output as expected
![image](https://github.com/google/generative-ai-docs/assets/59855919/4bc9c361-4424-4178-b267-82678b1f8f27)

### Any other information you'd like to share?

If we are using ! instead of %%bash , we may get indentation errors or probably no output
However when I ran the following command using subprocess, I got the output
![image](https://github.com/google/generative-ai-docs/assets/59855919/c1f339f2-48c7-4296-b5e5-71b669301dfc)

**Comment by MarkDaoust** [comment]
Yeah, those notebooks need to balance between clear outputs and understandable errors. 

The `2> /dev/null | grep text` is throwing out the stderr, and only showing the lines of stdout that contain `text` the errors should be a lot clearer if you remove that appendix.
